### PR TITLE
Rough draft of separate "Use Ethereum" page

### PR DESCRIPTION
**Pull request by jjmstark**
Creates a new "Use Ethereum" page, by separating out the "Use Ethereum Apps" content on the existing "Use" page. This is a quick and simple draft, which we will intend to iterate on.

See above.

### CODE REVIEW

File: docs/use_new/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# Using Ethereum {#using-ethereum}
+
+<div class="featured">
+
+  **Want to get started using Ethereum?** There are hundreds of decentralized applications built on ethereum that you can use today.
+
+</div>
+
+## Use an application built on ethereum {#1-use-an-application-built-on-ethereum}
```
Comment (samajammin) [inline docs/use_new/index.md]: `How to use an application (dApp) built on Ethereum`

File: docs/use_new/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Interested in looking at more applications built on Ethereum?
+
+- [Built on Ethereum](https://docs.ethhub.io/built-on-ethereum/built-on-ethereum/) *Updated often - EthHub*
+- [90+ Ethereum Apps You Can Use Right Now](https://media.consensys.net/40-ethereum-apps-you-can-use-right-now-d643333769f7) *April 24, 2019 - ConsenSys*
+- [Ethereum Dapps](https://www.stateofthedapps.com/rankings/platform/ethereum) *Updated often - State of the Dapps*
+- [Ethereum DeFi Ecosystem](https://defiprime.com/ethereum) *Updated often - Defiprime*
+- [DeFi Pulse](https://defipulse.com/) *Analytics + rankings of DeFi protocols - Defi Pulse*
+
+
+Some Ethereum applications require a wallet - [learn more about wallets here](https://www.ethereum.org/wallets).
```
Comment (samajammin) [inline docs/use_new/index.md]: `[learn more about Ethereum wallets here](/wallets)`

File: docs/use_new/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- [Built on Ethereum](https://docs.ethhub.io/built-on-ethereum/built-on-ethereum/) *Updated often - EthHub*
+- [90+ Ethereum Apps You Can Use Right Now](https://media.consensys.net/40-ethereum-apps-you-can-use-right-now-d643333769f7) *April 24, 2019 - ConsenSys*
+- [Ethereum Dapps](https://www.stateofthedapps.com/rankings/platform/ethereum) *Updated often - State of the Dapps*
+- [Ethereum DeFi Ecosystem](https://defiprime.com/ethereum) *Updated often - Defiprime*
+- [DeFi Pulse](https://defipulse.com/) *Analytics + rankings of DeFi protocols - Defi Pulse*
+
+
+Some Ethereum applications require a wallet - [learn more about wallets here](https://www.ethereum.org/wallets). 
+
+Some Ethereum applications will require ETH - [learn more about ETH here](https://www.ethereum.org/ETH).
```
Comment (samajammin) [inline docs/use_new/index.md]: `[learn more about ETH here](/eth)`

**Comment by samajammin** [comment]
@jjmstark question on terminology - `Dapps` vs. `dApps` vs. `DApp` - we use all of these across the site, I think we should pick one & try to stay consistent.

**Comment by jjmstark** [comment]
My strong view weakly held: we should just use "dapp" without any special capitalization. 

Reasons:

- no consistent standard has emerged in last few years, people still use all 3 of the versions you mention above
- assuming the term does enter mainstream usage, special capitalizations are likely to be discarded for convenience over time. There are few (if any?) words in the english language that have special capitalization rules (other than brand names). 
- "no special capitalization" is a natural schelling point

**Pull request merged**